### ISSUE TITLE
MotifDB button to create new motif sets from experiments that you own and can edit

### ISSUE THREAD

**Issue by justinjjvanderhooft** [sdrogers/ms2ldaviz#149]
Add button based on www.ms2lda.org/motifdb/create_motif_set etc

Only possible for experiments that you own and can edit.

Important: Block the example experiments?! (we have ones for GNPS and MassBank already!)

**Comment by joewandy** [comment]
Do you mean, add a button to go directly from an experiment (that the user owns) to creating motifdb based on that experiment?

**Comment by justinjjvanderhooft** [comment]
Yes, I think the link on the MotifDB page is clear enough now, but I think such a direct link from an experiment will help to promote the creation of more MotifSets.... What do you think?
